src/app/pages/categories/management/pages-transaction-categories.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import { map, Observable, Subject } from 'rxjs';
+import { DomainsTransactionCategoriesGateway } from 'src/app/domains/categories/domains.transaction-categories.gateway';
+import { TransactionCategory } from '../transaction-category.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class PagesTransactionCategoriesService {
+  private categories$: Subject<TransactionCategory[]> = new Subject<TransactionCategory[]>();
+
+  public constructor(
+    private readonly domainsTransactionCategoriesGateway: DomainsTransactionCategoriesGateway,
+  ) {}
+
+  public getCategories$ = (): Observable<TransactionCategory[]> => this.categories$.asObservable();
+
+  public getTransactionCategories(): void {
```

Comment:
`getTransactionCategories` and `getCategories$` look so similar I would assume they do the same. I would rename this one maybe to `loadCategories` and `getCategories$` to `getCategories` as we often add `$` to the observables not methods returning observables.